src/util.h

```diff
@@ -101,6 +101,9 @@ inline int CalcBitRateBitsPerSecFromCodedBytes ( const int iCeltNumCodedBytes,
 
 QString GetVersionAndNameStr ( const bool bWithHtml = true );
 QString MakeClientNameTitle ( QString win, QString client );
+#ifndef HEADLESS
```

Comment:
Similarly, the `GUI Utilities` section starts at line 371, so this should be added inside the existing `ifndef`, around line 497.